recipe/migrations/cuda102_110_111_112_ppc64le.yaml

```diff
@@ -0,0 +1,56 @@
+migrator_ts: 1645421738
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: true
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  wait_for_migrators:
+    - aarch64andppc64leaddition
+
+c_compiler_version:            # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
```

Comment:
Can you fix the versions to match x86_64?

Reply:
@isuruf 
Done. Kept 7 for cuda < 11 and 9 for cuda >= 11. Also changed cudnn versions. Please have a look.

Reply:
See https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L14-L17 for the versions that x86_64 uses.

Reply:
@isuruf 
Fixed the versions to match x86_64. Please have a look.